task3_13.c

```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+ 
+int main()
+{
+    const double A=4,B=6,C=13;
+    const double D=2,E=3,F=8;
+    const double det=A*E-B*D;
+    if(det == 0){
+      if(B*F == C*E){

```

Comment:
not the way to compare doubles

Reply:
fixed

Reply:
it is still not the way to compare double, use abs()<

Reply:
Have you checked these commits?
https://github.com/TymurHryhorian/CPPTasks/pull/3/commits/d087ad9b3a46f7cc77ea6e410b1a5ed3d14ab62c
https://github.com/TymurHryhorian/CPPTasks/pull/3/commits/5543e229f1f97f3d233f1869238d13fd17955469